src/browser/components/MainPage.jsx

```diff
@@ -296,6 +305,26 @@ const MainPage = React.createClass({
       authServerURL = `${tmpURL.protocol}//${tmpURL.host}`;
       authInfo = this.state.loginQueue[0].authInfo;
     }
+    var modal;
+    if (this.state.showNewTeamModal) {
+      modal = (
+        <NewTeamModal
```

Comment:
Possibly you can use `show` parameter of ReactBootstrap.Modal. In my memory, it would make better animation. Please see `LoginModal`.